plugins/device/init.go

```diff
@@ -22,11 +22,18 @@ func (p *Plugin) RestRoutes() route.Routes {
 		route.Route{
 			Name:         "DeviceAdd",
 			Method:       "POST",
-			Pattern:      "/peers/{peerid}/devices",
+			Pattern:      "/devices/{peerid}",
```

Comment:
`restclient` patch merged recently. please update this change in `$SRC/pkg/restclient/device.go`